### ISSUE TITLE
[4.0] [atum] Static danger alert contrast doesn't pass WCAG AA

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#29174]
### Steps to reproduce the issue

In the backend, add the following markup:

```
<div class="alert alert-danger">This is a danger alert</div>
```
This will render a danger alert. However the colour contrast does not meet the WCAG AA standards

![screeny](https://user-images.githubusercontent.com/2019801/82459950-779e3500-9ab0-11ea-84cf-60928a1e2449.png)

**Comment by hans2103** [comment]
- Joomla Administrator > Content > Site Modules
- Click New
- Select _Articles - Archived_
- Press _Save_

the alert box appears... but not in the markup as mentioned in this issue.

<img width="819" alt="Schermafdruk 2020-07-28 14 35 38" src="https://user-images.githubusercontent.com/639822/88665936-a99acb80-d0df-11ea-92ac-ad56467aa908.png">


The markup I see is:
```
<joomla-alert type="danger" dismiss="true" class="joomla-alert--show" role="alert"><button class="joomla-alert--close" aria-label="Close"><span aria-hidden="true">×</span></button><span class="alert-heading">Error</span><div>The form cannot be submitted as it's missing required data. <br> Please correct the marked fields and try again.</div></joomla-alert>
```

The code for this alert is added via layouts/joomla/system/message.php as a WebComponent
Styling is loaded from `./media/vendor/joomla-custom-elements/css/joomla-alert.min.css`
The styling for this alert is added via `node_modules/joomla-ui-custom-elements/src/scss/alert/alert/scss`

Changing the colors in Atum template does not change the colors of the alert boxes.
To change the color of the alert boxes we have to create a PR on https://github.com/joomla-projects/custom-elements


Can someone confirm this?

**Comment by C-Lodder** [comment]
@hans2103 The markup you've got is the custom element, where as I'm referring to the static Bootstrap alerts that are used in several places in Joomla, and potentially by 3rd party developers.

Inject the markup I've provided into your page and try again

**Comment by chmst** [comment]
@C-Lodder After the changes in the alerts which have beein made in #30400 and #30401, is this still an issue? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29174">issues.joomla.org/tracker/joomla-cms/29174</a>.</sub>

**Comment by C-Lodder** [comment]
@chmst Yes, it's still an issue. The changes on those PR's were for the alert custom element, not the static Bootstrap alert.

**Comment by brianteeman** [comment]
I can confirm the issue as reported by @C-Lodder 

I just have no idea how to resolve it.

**Comment by brianteeman** [comment]
Not sure when this was changed but this is no longer an issue and can be closed

![image](https://user-images.githubusercontent.com/1296369/128629540-3429807e-b6cf-4de9-8cd0-0978e2219949.png)

**Comment by drmenzelit** [comment]
Closing this as the contrast is ok
